cdap-docs/vars

```diff
@@ -46,10 +46,12 @@ GIT_BRANCH_CDAP_SECURITY_EXTN="release/0.2"
 GIT_BRANCH_CDAP_CLIENTS="1.2.0"
 GIT_BRANCH_CDAP_INGEST="1.3.0"
 
+# GIT_BRANCH_CDAP_APPS="release/cdap-3.6-compatible" # commented-out until available
```

Comment:
we generally don't keep commented code. I suggest we remove this line, and instead include this file in the release checklist, so release managers can update this when branches are available.